### ISSUE TITLE
Is there a way of running this on multiple threads?

### ISSUE THREAD

**Issue by sophiawisdom** [bertdobbelaere/SorterHunter#9]
Seems like it would be useful. I have for example a 60-core machine, so it would be 60x faster. I haven't looked much at the code, but I would be delighted at a version that just runs all the threads independently and syncs every time there's a new best network.

**Comment by bertdobbelaere** [comment]
Currently the SorterHunter program itself is still single threaded. Because the instances don't have to interact, a simple divide-and-conquer approach is to launch multiple program instances and to process the results in an automated way using a launcher script. This is what I did in the past using the Python scripts I just added in the FarmManager directory. The main advantage of the approach is that it can readily span multiple machines without extra programming effort (and I don't have to change the C++ program for it :-) ).

There is something to say to add a "native" parallelisation feature to SorterHunter itself (multithreading), but I'm a bit afraid portability would suffer, and it would be complex to manage multiple machines. I like to keep the main program simple and understandable, focusing on the networks.
